specs/UserDataFolder.md

```diff
@@ -0,0 +1,102 @@
+
+Title
+===
+UserDataFolder API 
+
+# Background
+You may have a need to cleanup or possibly add to the data being
+stored in the WebView2 user data directory.  This directory can be either 
+configured by you during creation of the webview2 control or calculated
+at runtime by the control if it wasn't set during creation.
+
+The current WebView2 code is designed to use a the directory that the hosting 
+application is running in as the default location.  This has resulted in issues 
+if running from a folder for which the WebView2 process doesn't have write access 
+such as a Program Files folder.  Work is in progress to change that default and 
+will be forthcoming in later changes.
+
+With the current guidance it is possible for you to have built
```

Comment:
```suggestion
With the current implementation it is possible for you to have built
```

Reworded. I don't think "use the default" is our guidance, especially given the problematic default for TrustedInstaller applications.